packages/shared/src/components/PostOptionsMenu.tsx

```diff
@@ -350,13 +350,17 @@ export default function PostOptionsMenu({
     postOptions.push({
       icon: (
         <MenuIcon
-          Icon={sourceSubscribe.isSubscribed ? BellSubscribedIcon : BellIcon}
+          Icon={
+            sourceSubscribe.haveNotifications ? BellSubscribedIcon : BellIcon
+          }
         />
       ),
       label: `${
-        sourceSubscribe.isSubscribed ? 'Unsubscribe from' : 'Subscribe to'
+        sourceSubscribe.haveNotifications
+          ? 'Receive notifications from'
+          : 'Receive notifications to'
       } ${post?.source?.name}`,
-      action: sourceSubscribe.isReady ? sourceSubscribe.onSubscribe : undefined,
+      action: sourceSubscribe.isReady ? sourceSubscribe.onNotify : undefined,
```

Comment:
Should this and related changes also be behind experiment? To preserve control copy.

Reply:
Yes, I've asked for updates on this component [here](https://dailydotdev.slack.com/archives/C07ATUPBPJ9/p1721396757912089) and they have updated the UI early today. I've to update this!